Library/Homebrew/dev-cmd/update-python-resources.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  module_function
+
+  def update_python_resources_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `update-python-resources` [<options>] <formula>
+
+        Update versions for PyPI resource blocks in <formula>.
+      EOS
+      switch "-p", "--print-only",
+             description: "Print the updated resource blocks instead of changing the formula."
+      named :formula
+    end
+  end
+
+  def update_python_resources
+    update_python_resources_args.parse
```

Comment:
```suggestion
    args = update_python_resources_args.parse
```
See https://github.com/Homebrew/brew/pull/8068

Reply:
Good catch, thanks!

Reply:
It looks like this needs to wait for #8068

Reply:
Should be working now.

Reply:
@reitermarkus should all commands follow this style now? I see that you added `args =` to several commands in #8068 but not in all (`bump-formula-pr` didn't get this change, for example). Should it be changed as well?